stories/src/examples/api/getDataSelection.ts

```diff
@@ -1,35 +1,54 @@
 import { LatLngTuple } from 'leaflet'
 
-const generateParams = (data: { [key: string]: any }) =>
+const generateParams = (data: Record<string, string>) =>
   Object.entries(data)
     .filter(([, value]) => value !== null && value !== undefined)
     .map((pair) => pair.map(encodeURIComponent).join('='))
     .join('&')
 
-export type DataSelection = {
+export interface Source {
+  centroid: [number, number]
+}
+
+export interface ObjectListItem {
+  _index: string
+  _type: string
+  _id: string
+  _score: null
+  _source: Source
+  sort: Array<number | string>
```

Comment:
Is it necessary to include these types? Looks like they're not used.

Reply:
For completion sake I included them, don't see the harm.